.github/actions/install-nightly-ice/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Instal Nightly Ice
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install Ice Brew Formula
+      if: runner.os == 'macOS'
+      shell: bash
+      run: |
+        brew tap zeroc-ice/nightly  https://download.zeroc.com/nexus/repository/nightly/homebrew-nightly.git
+        brew install zeroc-ice/nightly/ice
+
+    - name: Install Ice APT Repository
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        sudo mkdir -p /etc/apt/keyrings
+        sudo curl -fsSL https://download.zeroc.com/GPG-KEY-zeroc-nightly | sudo gpg --dearmor -o /etc/apt/keyrings/zeroc-nightly.gpg
+        echo "deb [signed-by=/etc/apt/keyrings/zeroc-nightly.gpg] https://download.zeroc.com/nexus/repository/ubuntu24.04-nightly nightly main" | sudo tee /etc/apt/sources.list.d/zeroc-nightly.list
+
+        sudo apt-get update
+        sudo apt-get install zeroc-ice-all-dev
```

Comment:
don't we need to pass `-y` for non interactive mode?

Reply:
I think they set some env variables so you don't need do but I think it's better to add it.